### ISSUE TITLE
Issues when resuming game mode from saved game

### ISSUE THREAD

**Issue by jfsalaza** [CaptainPStar/co10_Escape#101]
As mentioned in the title, resuming from a save state, whether after the whole party died, or because we intend to resume the game later exhibits the following issues:

1. Everyone except the host loses their weapons
2. Everyone except the host can't grab certain enemy equipment such as enemy uniforms, or items inside enemy uniforms
3. When you resume from one of the towers, everyone except the host spawns outside of the towers if the a save state was created inside the tower.

I recorded some of those issues while playing the mode, let me know which is the best way to share those clips so it can be used for debugging.

Thank you.

**Comment by NeoArmageddon** [comment]
Thank you for the report. Saving is not really supported as there are too many scripts that are not stateless and which will result in faulty execution after loading a save. Arma has a history of MP saving not really working except with the simplest missions.

When I have some time I may come up with a solution but right now my suggestion is:
Play Escape in one session as saving is not supported.

**Comment by jfsalaza** [comment]
Thank you for answering!

Something I forgot to add is that in one occasion, resuming from saving kinda bugged out and I in particular (non host) had to leave the game and rejoin. When I respawned I did so as some sort of special forces and had equipment (guns, uniform, helmet, etc..). I am not sure what triggered that and I unfortunately do not have a clip of that situation. 

Maybe adding a game option that can be set before starting the game that gives non-host users basic equipment when starting from a save state could be a way to circumvent that.

Thanks!
